proposals/NNNN-close-app.md

```diff
@@ -0,0 +1,63 @@
+# CloseApp RPC
```

Comment:
Should the RPC be called `CloseApplication` as it is described in the `Proposed solution` section?